client/components/wp-login-footer/index.jsx

```diff
@@ -0,0 +1,68 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { getCurrentOAuth2Client } from 'state/ui/oauth2-clients/selectors';
+import JetpackColophon from 'components/jetpack-colophon';
+
+class LoginFooter extends Component {
```

Comment:
If this component is `LoginFooter`, its directory should match, i.e. `login-footer` not `wp-login-footer`.